accounts/abi/bind/backends/blockchain.go

```diff
@@ -301,7 +301,7 @@ func (b *BlockchainContractBackend) FilterLogs(ctx context.Context, query kaia.F
 	if !ok {
 		return nil, errors.New("BlockChainForCaller is not blockchain.BlockChain")
 	}
-	filter := filters.NewRangeFilter(&filterBackend{state.Database().TrieDB().DiskDB(), bc}, from, to, query.Addresses, query.Topics)
+	filter := filters.NewRangeFilter(&filterBackend{state.Database().TrieDB().DiskDB(), bc, nil}, from, to, query.Addresses, query.Topics)
```

Comment:
Can `nil` backend cause panic in `filterBackend.Pending()`?

Reply:
This is a one-time instantiation, and we won't encounter a Panic unless we use `filterBackend.Pending()` on `NewRangeFilter `. And I modified APIs not to support `pending logs` in this time.